app.server.js

```diff
@@ -13,6 +13,8 @@ const HOST = "0.0.0.0";
 const GOOGLE_CREDENTIAL =
   process.env.GOOGLE_CREDENTIAL ||
   "990846956506-bfhbjsu4nl5mvlkngr3tsmfcek24e8t8.apps.googleusercontent.com";
+const ENFORCE_SSL = process.env.ENFORCE_SSL || false;
```

Comment:
Local testing didn't works, I think you need compare environment variables with strings

```suggestion
const ENFORCE_SSL = process.env.ENFORCE_SSL === 'true';
```